src/main/java/org/broadinstitute/consent/http/models/dataset_registration_v1/builder/ConsentGroupFromDataset.java

```diff
@@ -40,7 +25,7 @@
 public class ConsentGroupFromDataset {
 
   public ConsentGroup build(Dataset dataset) {
-    if (Objects.nonNull(dataset)) {
+    if (dataset != null) {
```

Comment:
is there a reason for this change? there is a use of nonNull below on that is unchanged. Do we want to avoid using nonNull?

Reply:
Yes, we've been moving in this direction for a while, originally based on Phil's observation that the use of `Objects.**` is meant for use in streaming operations.